pages/Product.vue

```diff
@@ -404,7 +404,7 @@ export default {
           qty: this.getCurrentProduct.qty
         })
 
-        this.manageQuantity = res.isManageStock
+        this.manageQuantity = !!res.isManageStock
```

Comment:
I think it would be better to use explicit way of casting with `Boolean(res.isManageStock)` - it should be understandable for a bigger part of the community.

Reply:
Yup! I've created a new commit: https://github.com/DivanteLtd/vsf-default/pull/27/commits/742c1b5e214f39f32fc4aba9003ec9c8d3be6400